### PR TITLE
Docs/admin ui personal settings

### PR DESCRIPTION
**Pull request by gallayl**

### CODE REVIEW

**Review by zoltanbedi — APPROVED** [review]

File: _docs/admin-ui-personal-settings.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Language
+
+You can select between the `default` (english) and `hungarian` language with the `language` field.
+
+## Events and logging
+
+You can define the size of the event log with the `eventLogSize`. Only the last N entries will be saved. There is another setting for the entries to persist in the log: the `logLevel` field. You can also enable / disable attaching the log when you send a crash report with the `sendLogWithCrashReports` field.
+
+![Changing the theme in the personal settings](/img/admin-ui-personal-setting-theme.gif "Changing the theme in the personal settings")
+admin-ui-personal-setting-theme
```
Comment (zoltanbedi) [inline _docs/admin-ui-personal-settings.md]: For me this feels weird in the site. (line 61)

File: _docs/admin-ui-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+You can define a custom query with results using the Query widget. The `query` field should contain a content query, you can define the `columns` and define some controls to be shown or hidden. The fields are: `showColumnNames` (this also enables ordering), `showOpenInSearch` and `showRefresh`.
+
+The query widget has an additional mode that you can enable with the `countOnly` field - if you enable this, only the hit count will be displayed.
+
+## Updates widget
+
+This one will check the _current repository's_ version info and compares it with the latest component versions published to [nuget.org](https://nuget.org). It will notify you when updates are available.
+
+![The default dashboard](/img/admin-ui-default-dashboard.png "The default dashboard")
```
Comment (herflis) [inline _docs/admin-ui-dashboard.md]: Maybe a few words (list of widgets) would be nice about the default dashboard. :nerd_face:

File: _docs/admin-ui/personal-settings.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+## Language
+
+You can select between the `default` (english) and `hungarian` language with the `language` field.
+
+## Events and logging
+
+You can define the size of the event log with the `eventLogSize`. Only the last N entries will be saved. There is another setting for the entries to persist in the log: the `logLevel` field. You can also enable / disable attaching the log when you send a crash report with the `sendLogWithCrashReports` field.
+
+![Changing the theme in the personal settings](/img/admin-ui-personal-setting-theme.gif "Changing the theme in the personal settings")
+admin-ui-personal-setting-theme
```
Comment (herflis) [inline _docs/admin-ui/personal-settings.md]:
```suggestion
```

File: _docs/admin-ui-personal-settings.md (hunk 1 above)
Comment (gallayl) [reply _docs/admin-ui-personal-settings.md]: 🍰

**Review by herflis — APPROVED** [review]
:owl:

**Pull request merged**